qiskit_ibm_runtime/runtime_job.py

```diff
@@ -666,7 +666,7 @@ def usage_estimation(self) -> Dict[str, Any]:
         Returns:
             ``quantum_seconds`` which is the estimated quantum time
```

Comment:
References to quantum seconds need to be changes as well, but @jyu00 is opening a second PR for this.